lib/tasks/users.rake

```diff
@@ -48,6 +48,9 @@ namespace :db do
       i += rand(10 ** 7)
       student.update_attribute(:id_number, sprintf('%010d', i))
       student.update_attribute(:grace_credits, 5)
+      if student.id == 10
+        student.update_attribute(:email, 'ignaspan@gmail.com')
```

Comment:
You should revert this change as well (just for testing purposes).

Reply:
Oh, yes and make a PR for emails